src/documents/models.py

```diff
@@ -286,6 +286,10 @@ def find_renamed_document(self, subdirectory=""):
         # Go up in the directory hierarchy and try to delete all directories
         root = os.path.normpath(Document.filename_to_path(subdirectory))
 
+        # Check if root really exists and return otherwise
+        if not os.path.isdir(root):
+            return None
```

Comment:
I'm a little confused here. Why is that necessary? Why didn't we pick up on this earlier? Could you explain a little - in the commit-message, if possible.

Reply:
Turns out, that override_settings is actually keeping the override for
    following test cases (even if you disable them). This leads to an
    overriden MEDIA_ROOT that no longer exists. test_file_deletion does not
    do any file operations, but find_renamed_document was assuming the
    MEDIA_ROOT is actually exist.

This is actually pretty ugly and I am open for better suggestions.